web/modules/custom/cyber_breadcrumbs/src/BasicBreadcrumb.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Drupal\cyber_breadcrumbs;
+
+/**
+ * @file
+ * Generates the breadcrumb trail for content including:
+ * - Publication
+ *
+ * In the format:
+ * > Home
+ * > current-page-title
+ *
+ * > <front>
+ * > /current-page-title
+ */
+use Drupal\Core\Breadcrumb\Breadcrumb;
+use Drupal\Core\Breadcrumb\BreadcrumbBuilderInterface;
+use Drupal\Core\Controller\TitleResolverInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+use Drupal\node\NodeInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class BasicBreadcrumb implements BreadcrumbBuilderInterface {
+
+  /**
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+  /**
```

Comment:
New line to separate member variables from the next comment:

```
/**
   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
   */
  protected $entityTypeManager;
/**
```
should look like:
```
/**
   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
   */
  protected $entityTypeManager;

/**
 * etc etc
```

PHPCS might bark about this but if not it's how things are formatted in general and makes it more readable.